android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -1221,6 +1222,11 @@ public boolean canShowToast(int stringRes, @Nullable CharSequence text) {
                     Boolean quickBookmark = configJson.getBoolean(KEY_CONFIG_QUICK_BOOKMARK_CREATION);
                     builder.quickBookmarkCreation(quickBookmark);
                 }
+                if (!configJson.isNull(KEY_CONFIG_FULL_SCREEN_MODE_ENABLED)) {
```

Comment:
hmm is this actually default to false? if user don't set it, we don't call this, in native viewer we default to true, so we need to manually default to false if this is not set